Orange/canvas/application/addons.py

```diff
@@ -300,6 +300,7 @@ def set_items(self, items):
 
             model.appendRow([item1, item2, item3, item4])
 
+        model.sort(1)  # sorting list of add-ons in the alphabetical order
```

Comment:
It does not work correctly. Have you tested if then the correct add-ons are also installed? If I added "pip" to the list and tried to upgrade it, it upgraded the wrong package.

Reply:
@PrimozGodec Try `self.__proxy.sort(1)`

Reply:
Thank you @markotoplak and @ales-erjavec. I think that now it works.